test/unit/specs/components/Identity/createComponent.spec.js

```diff
@@ -30,13 +35,23 @@ describe("Identity::createComponent", () => {
     );
     handleResponseForIdSyncs = jasmine.createSpy("handleResponseForIdSyncs");
     getEcidFromResponse = jasmine.createSpy("getEcidFromResponse");
+    getEcidPromise = defer();
```

Comment:
I recommend calling this `getEcidDeferred` since it's a deferred and not a promise. It would also be consistent with `consentDeferred`.